### ISSUE TITLE
Port open to connect to Kasm App

### ISSUE THREAD

**Issue by githubja-lab** [kasmtech/workspaces-issues#250]
Hi, I am wondering if I can take a custom container and established the communication with KasmApp using VNC. I am not using the core-images from kasmweb.

**Comment by j-travis** [comment]
No, not really.

Kasm Workspaces is currently built around connecting to containers that have KasmVNC baked in and configured in a particular way. Our Core images provide that baseline of functionality.

Can you explain a bit more about your use case in case it can be accomplished in a different way

**Comment by githubja-lab** [comment]
Hi Travis, 
I am custom building images (Rocky and Mac OS) and I will like to use Kasm as the display, I able to customize the VNC port forwarding to enable access to images for example I can VNC using any VNC client to the container with no problem so I was wondering if I can take this to the next level and handling the docker run to Kasm and the streaming gui to the web base  instead of having to launch an extra instace ( ex remmina) to open toe container.

**Comment by j-travis** [comment]
In workspaces 1.12, we added the ability to connect to arbitrary machines running RDP or KasmVNC. We extended that in 1.13 to also support connecting to machines running SSH or VNC.

https://kasmweb.com/docs/latest/how_to/fixed_infrastructure.html#vnc

So hopefully you now have multiple options to connect to your other machines.

Wel also no have rocky docker worksaces: https://hub.docker.com/r/kasmweb/rockylinux-8-desktop
